app/assets/javascripts/discourse/app/helpers/slow-mode.js

```diff
@@ -0,0 +1,67 @@
+import I18n from "I18n";
```

Comment:
moment js has a lot of helpers to deal with duration, maybe you should have a look at it ?